website/blog/2020-07-20-docs-update.md

```diff
@@ -0,0 +1,36 @@
+---
+title: React Native Documentation Update
+author: Rachel Nabors
+authorTitle: Documentation Engineer at Facebook
+authorURL: https://twitter.com/rachelnabors
+authorImageURL: https://avatars0.githubusercontent.com/u/236306?s=460&v=4
+authorTwitter: rachelnabors
+---
+
+Last year we conducted user interviews and sent out [a survey](https://www.surveymonkey.co.uk/r/DDZWQDJ) to learn more about how and when people use these React Native docs. With the data and guidance gleaned from 24 interviews and over 3000 survey responses, we've been able to work to improve React Native's documentation, and we're excited to share that progress today:
+
+- **[New Getting Started guides](https://fb.workplace.com/groups/529650227572836/permalink/656344974903360/)** We launched new Getting Started with React Native docs to [explain what Native Components](https://reactnative.dev/docs/intro-react-native-components) are to people with no mobile development background. We also included a [refresher/introduction to React](https://reactnative.dev/docs/intro-react) to help folks getting started with React for the first time!
```

Comment:
```suggestion
- **[New Getting Started guides](https://fb.workplace.com/groups/529650227572836/permalink/656344974903360/)** We launched new Getting Started with docs to [explain what Native Components](https://reactnative.dev/docs/intro-react-native-components) are to people with no mobile development background. We also included a [refresher/introduction to React](https://reactnative.dev/docs/intro-react) to help folks getting started with React for the first time!
```

Would drop React Native since what else would the docs be for?